assets/scripts/fe/package.json

```diff
@@ -3,11 +3,12 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
-    "build": "ng build",
+    "start": "ng serve & ngtw watch",
```

Comment:
@xaviertactiv do not merge your tailwind packages with angular. create your own npm inside `assets/`. 

`assets/scripts/fe` <-- angular files